main.go

```diff
@@ -29,7 +29,7 @@ type configuration struct {
 	HealthCheckPath       string   `env:"HEALTH_CHECK_PATH,default=/en-US/static/html/credit.html"` // Health Check path in splunk, this path is proxied w/o auth. The default is a static file served by the splunk web server
 	EmailSuffix           string   `env:"EMAIL_SUFFIX,default=@heroku.com"`                         // Required email suffix. Emails w/o this suffix will not be let in
 	StateToken            string   `env:"STATE_TOKEN,required"`                                     // Token used when communicating with Google Oauth2 provider
-  ServerHost            string   `env:"HOST"`                                                     // Host
+        ServerHost            string   `env:"HOST"`                                                     // Host
```

Comment:
Here's an example of setting up vim for go formatting
https://github.com/jmervine/vimrc/blob/master/filetypes.vim#L32-L34

Here's setting up go plugin's if you're using vundle
https://github.com/jmervine/vimrc/blob/master/bundles.vim#L85-L87

Otherwise, you need to run `go fmt` before you commit.

Reply:
yes got it, thanks